perfkitbenchmarker/linux_packages/openjdk_msft.py

```diff
@@ -0,0 +1,45 @@
+# linux_packages/openjdk_msft.py
```

Comment:
Styling nit: Can you add the usual copyright block (see another file, e,g, aerospike_client.py, for example. you just need to update the year) at the top?
After that, add a docstring such as
"""Module to install Microsoft's OpenJDK package."""

Reply:
I am going to pull this change in for now. Feel free to open another pull request to add the docstring and copyright block.